bip-internalkey.md

```diff
@@ -0,0 +1,68 @@
+| BIP-XXXX   | `OP_INTERNALKEY`
+| :--------- | :---------------
+| Layer      | Consensus (soft fork)
+| Author     | Brandon Black `<freedom@reardencode.com>`, Jeremy Rubin `<j@rubin.io>`
+| Discussion | https://delvingbitcoin.org/t/lnhance-bips-and-implementation/376/9
+| Status     | Draft
+| Type       | Standards Track
+| Created    | 2023-12-22
+| License    | BSD-3-Clause
+
+## Abstract
+
+This BIP describes a new tapscript opcode (`OP_INTERNALKEY`) which
+pushes the taproot internal key to the stack.
+
+## Specification
+
+When verifying taproot script spends having leaf version `0xc0` (as defined in
+[BIP 342]), `OP_INTERNALKEY` replaces `OP_SUCCESS203` (0xcb). `OP_INTERNALKEY`
+pushes the taproot internal key, as defined in [BIP 341], to the stack.
+
+## Motivation
```

Comment:
Additional useful feature: ability to inspect the tweak to a key (in combination with some OP_TWEAKVERIFY).

Reply:
That would require additional introspection beyond just INTERNALKEY too, wouldn't it?

Reply:
Yes, something like TWEAKADD or TWEAKVERIFY.